### DISCUSSION TITLE
Webflasher only with github login

### DISCUSSION THREAD

**Discussion by eh2k** [eh2k/squares-and-circles#112] [General]
In case you are wondering why you now have to log in for the Webflasher.

I would like to use the infrastructure of github (github API) more in the future.

The Github API is quite handy to list the available versions, or to access the compiled engine files, that stored in the repo.
My goal is also to use the github actions to compile the FV1 effects or similar (work in progress).

I think that users without a Github account might also have a reason to sign up. And they would be more likely to report any issues or give feedback in general.

**Comment by N-EMAND** [comment]
Hi there,

i wanted to update my device, to the newest software. Poorly, the installation fails everytime. now i have only a ready to update device... anyone, could help maybe?
<img width="572" alt="image" src="https://github.com/user-attachments/assets/e7bc6678-a975-4b76-9a7b-62f9819f3314" />

**Comment by eh2k** [comment]
@N-EMAND 

Probably the website does not have permission to access WebMidi, please check this:

![image](https://github.com/user-attachments/assets/edde7b44-9cf2-4816-9a9a-72e855e09c3e)

**Reply by N-EMAND** [reply]
hey thank you for fast response.

i think the WebMidi is Ok
<img width="628" alt="image" src="https://github.com/user-attachments/assets/95b38c8f-a931-4f3c-a43c-bc8e52e073ed" />

**Reply by N-EMAND** [reply]
I Think the problem is fixed. Options in Browser prevent the access to update the device.

thank you very much

**Reply by KoenTh** [reply]
What were these options? Update is not working for me qlthough webmidi seems to be working. No Update possibility there though; so I only have the GND machine.
